pkg/nfd-gc/nfd-gc.go

```diff
@@ -37,6 +38,10 @@ import (
 	"sigs.k8s.io/node-feature-discovery/pkg/version"
 )
 
+const (
+	OwnerPod = "owner-pod"
```

Comment:
I think the label should have a prefix

```suggestion
	NRTOwnerPodLabel = "nfd.node.kubernetes.io/owner-pod"
```